airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -374,47 +558,82 @@ definitions:
         type: string
         enum: [DatetimeBasedCursor]
       cursor_field:
-        description: The location of the value on a record that will be used as a bookmark during sync
+        title: Cursor Field
+        description: The location of the value on a record that will be used as a bookmark during sync.
         type: string
+        examples:
+          - "created_at"
+          - "{{ config['record_cursor'] }}"
       datetime_format:
-        description: The format of the datetime
+        title: Datetime Format
+        description: The format of the datetime value.
         type: string
+        examples:
+          - "%Y-%m-%dT%H:%M:%S.%f%z"
       cursor_granularity:
-        description: Smallest increment the datetime_format has (ISO 8601 duration) that is used to ensure the start of a slice does not overlap with the end of the previous one
+        title: Cursor Granularity
+        description:
+          Smallest increment the datetime_format has (ISO 8601 duration) that is used to ensure the start of a slice does not overlap with the end of the previous one, e.g. for %Y-%m-%d the granularity should
+          be P1D, for %Y-%m-%dT%H:%M:%SZ the granularity should be PT1S.
         type: string
+        examples:
+          - "PT1S"
       end_datetime:
-        description: The datetime that determines the last record that should be synced
+        title: End Datetime
+        description: The datetime that determines the last record that should be synced.
         anyOf:
           - type: string
           - "$ref": "#/definitions/MinMaxDatetime"
+        examples:
+          - "2021-01-1T00:00:00Z"
```

Comment:
Add an example with "now" here as it's quite common?

Reply:
added examples for now_utc and now_local